Cubical/Data/Int/MoreInts/QuoInt/Properties.agda

```diff
@@ -233,3 +235,82 @@ sign-pos-suc-· m (signed s (suc n)) = refl
 
 ·-injʳ : ∀ m n o → m · pos (suc o) ≡ n · pos (suc o) → m ≡ n
 ·-injʳ m n o p = ·-injˡ o m n (·-comm (pos (suc o)) m ∙ p ∙ ·-comm n (pos (suc o)))
+
+opIsoIntℤ : Iso (Int → Int → Int) (ℤ → ℤ → ℤ)
+opIsoIntℤ = (opIso isoIntℤ)
```

Comment:
Extra pair of parentheses, also this is so short that this can probably be inlined for a bit less cognitive indirection.

Reply:
Yeah, I'm not sure why I did that anyway, inlined.